tests/qiskit_backend_test.py

```diff
@@ -122,7 +123,7 @@ def test_qiskit_counts(belem_emulator_backend: IBMQEmulatorBackend) -> None:
     qc = QuantumCircuit(num_qubits)
     qc.h(0)
     qc.cx(0, 1)
-    circfn = CircuitStateFn(qc)
+    circfn = Statevector(qc)
 
     s = CircuitSampler(TketBackend(belem_emulator_backend))
 
```

Comment:
In the testcases that I am running locally this line breaks with
```belem_emulator_backend = <pytket.extensions.qiskit.backends.ibmq_emulator.IBMQEmulatorBackend object at 0x7f8565019c60>

    @pytest.mark.skipif(skip_remote_tests, reason=REASON)
    def test_qiskit_counts(belem_emulator_backend: IBMQEmulatorBackend) -> None:
        num_qubits = 2
        qc = QuantumCircuit(num_qubits)
        qc.h(0)
        qc.cx(0, 1)
        circfn = Statevector(qc)
    
        s = CircuitSampler(TketBackend(belem_emulator_backend))
    
>       res = s.sample_circuits([circfn])

qiskit_backend_test.py:130: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <qiskit.opflow.converters.circuit_sampler.CircuitSampler object at 0x7f85651c6f80>
circuit_sfns = [Statevector([0.70710678+0.j, 0.        +0.j, 0.        +0.j,
             0.70710678+0.j],
            dims=(2, 2))], param_bindings = None

    def sample_circuits(
        self,
        circuit_sfns: Optional[List[CircuitStateFn]] = None,
        param_bindings: Optional[List[Dict[Parameter, float]]] = None,
    ) -> Dict[int, List[StateFn]]:
        r"""
        Samples the CircuitStateFns and returns a dict associating their ``id()`` values to their
        replacement DictStateFn or VectorStateFn. If param_bindings is provided,
        the CircuitStateFns are broken into their parameterizations, and a list of StateFns is
        returned in the dict for each circuit ``id()``. Note that param_bindings is provided here
        in a different format than in ``convert``, and lists of parameters within the dict is not
        supported, and only binding dicts which are valid to be passed into Terra can be included
        in this list.
    
        Args:
            circuit_sfns: The list of CircuitStateFns to sample.
            param_bindings: The parameterizations to bind to each CircuitStateFn.
    
        Returns:
            The dictionary mapping ids of the CircuitStateFns to their replacement StateFns.
        Raises:
            OpflowError: if extracted circuits are empty.
        """
        if not circuit_sfns and not self._transpiled_circ_cache:
            raise OpflowError("CircuitStateFn is empty and there is no cache.")
    
        if circuit_sfns:
            self._transpiled_circ_templates = None
>           if self._statevector or circuit_sfns[0].from_operator:
E           AttributeError: 'Statevector' object has no attribute 'from_operator'. Did you mean: 'to_operator'?

site-packages/qiskit/opflow/converters/circuit_sampler.py:295: AttributeError

Reply:
My understanding is that  Statevector(qc) is not a valid parameter for the `sample_circuits([circfn])` function?

Reply:
I think this needs one more step to get it to work, see https://qiskit.org/documentation/migration_guides/opflow_migration.html#state-functions

Reply:
Hello @cqc-melf, thank you for your message. I am currently investigating the issue. 
The error went unnoticed because the test was skipped.  
After running `pytest qiskit_backend_test.py -r s`, I noticed that the required IBM configuration is missing, resulting in the following message:
```
PYTKET_RUN_REMOTE_TESTS not set (requires IBM configuration)
```
I will address this issue and use the link you sent to resolve the error you mentioned.  Thank you.

Reply:
Hello, @cqc-melf. I was able to fix this.
I did not use Statevector(), I  did the following:
- defined a `sampler = BackendSampler(backend=backend)`
- passed the circuit to the sampler by using `sampler.run(qc).`
- extracted the number of qubits via `sampler.circuits[0].num_qubits`

I will push the changes and update the description.